src/wx/panel.cpp

```diff
@@ -1476,7 +1476,7 @@ class FilterThread : public wxThread {
     // if NULL, end thread
     uint8_t* src;
 
-    ExitCode Entry()
```

Comment:
It's like the return value of a command in the shell, since a thread is essentially a process, 0 is success and anything else is failure.

http://docs.wxwidgets.org/trunk/classwx_thread.html#af50fa7d3ac55002e367e79989b9cbc5d

This is the signature of the pure virtual method, if you change it won't be called. The return type has to match the API.

Also the difference between `void` and `void*` in C and C++ is that the former means no return value, while the latter means 'any kind of pointer'.

This needs to be fixed differently, check the logic of the function and make sure in every logical branch there is a return statement, `0` for success and e.g. `1` for failure.